src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -197,6 +197,7 @@ protected Object[][] getContents() {
         {"R_lastUpdateCountPropertyDescription", "Ensures that only the last update count is returned from an SQL statement passed to the server."},
         {"R_disableStatementPoolingPropertyDescription", "Disables the statement pooling feature."},
         {"R_integratedSecurityPropertyDescription", "Indicates whether Windows authentication will be used to connect to SQL Server."},
+        {"R_useDefaultGSSCredentialPropertyDescription", "Indicates whether should GSSCredential should be created using native GSS-API."},
```

Comment:
awkward, please rephrase

Reply:
Indicates whether GSSCredential will be created using native GSS-API.
```suggestion
        {"R_useDefaultGSSCredentialPropertyDescription", "Indicates whether GSSCredential will be created using native GSS-API."},
```